ci: ubuntu: xdpyinfo not found

Symptom on GitHub Actions Ubuntu:
```
Run bb test ide --platform=jvm --browser=firefox --launch-virtual-display

[ TASK test ide --platform=jvm --browser=firefox --launch-virtual-display ]------

[ Launching virtual display ]----------------------------------------------------
----- Error --------------------------------------------------------------------
Type:     java.io.IOException
Message:  Cannot run program "xdpyinfo": error=2, No such file or directory

Error: Process completed with exit code 1.
```

Looks like we were hit by: https://github.com/actions/virtual-environments/issues/5894

I follow up with the recommended fix.

--- a/.github/workflows/test.yml
+++ b/.github/workflows/test.yml
@@ -62,7 +62,7 @@ jobs:
 
     - name: Install Ubuntu X11 Utils
       if: ${{ matrix.os == 'ubuntu' }}
-      run: sudo apt install x11-utils
+      run: sudo apt-get -y install x11-utils
 
     - uses: actions/checkout@v3
 

--- a/.github/workflows/test.yml
+++ b/.github/workflows/test.yml
@@ -60,6 +60,10 @@ jobs:
       if: ${{ matrix.os == 'windows' }}
       run: Disable-NetAdapterChecksumOffload -Name * -TcpIPv4 -UdpIPv4 -TcpIPv6 -UdpIPv6
 
+    - name: Install Ubuntu X11 Utils
+      if: ${{ matrix.os == 'ubuntu' }}
+      run: sudo apt install x11-utils
+
     - uses: actions/checkout@v3
 
     - name: Clojure deps cache

